docs/guides/api-rest/menu.json

```diff
@@ -0,0 +1,10 @@
+{
+  "title": "API(REST)",
```

Comment:
We should call it "API (Rest)" or make it consistent with the other sections. (Lib, CLI etc.)

Reply:
I believe the proper casing is REST